misc/python/materialize/git.py

```diff
@@ -183,6 +183,8 @@ def try_get_remote_name_by_url(url: str) -> str | None:
         remote, desc = line.split("\t")
         if desc.lower() == f"{url} (fetch)".lower():
```

Comment:
Might this be clearer using [`removesuffix`](https://docs.python.org/3/library/stdtypes.html#str.removesuffix)? e.g. `desc.removesuffix(".git").lower() == f"{url} (fetch)".lower()`?